src/SignalRServiceExtension/TriggerBindings/Resolver/DefaultRequestResolver.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Buffers;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+using Microsoft.AspNetCore.SignalR.Protocol;
+using Microsoft.Azure.SignalR.Serverless.Protocols;
+
+namespace Microsoft.Azure.WebJobs.Extensions.SignalRService
+{
+    internal class DefaultRequestResolver : IRequestResolver
```

Comment:
>DefaultRequestResolver [](start = 19, length = 22)

Will IRequestResolver be public? If not, probably rename this class to SignalRRequstResolver is better.

Reply:
Updated